keyboards/sawnsprojects/plaque80/info.json

```diff
@@ -0,0 +1,148 @@
+{
+    "keyboard_name": "Plaque80",
+    "manufacturer": "SawnsProjects",
+    "url": "",
+    "maintainer": "qmk",
+    "usb": {
+        "vid": "0x5350",
+        "pid": "0x0801",
+        "device_version": "1.0.0"
+    },
+    "processor": "atmega32u4",
+    "bootloader": "atmel-dfu",
+    "matrix_pins": {
+        "rows": ["D3", "D5", "B1", "B2", "B3", "B7", "D1", "D0", "F4", "F5", "F6", "F7"],
+        "cols": ["F1", "C7", "C6", "B6", "B5", "B4", "D7", "D6", "D4"]
+    },
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "mousekey": true,
+        "extrakey": true,
+        "console": false,
+        "command": false,
+        "nkro": false,
+        "backlight": false,
+        "rgblight": true,
+        "audio": false,
+        "key_lock": false
+    },
+    "rgblight": {
+        "pin": "F0",
+        "led_count": 0,
```

Comment:
Zero LEDs, either the rgblight config should be removed or this value corrected.